src/urllib3/connectionpool.py

```diff
@@ -556,7 +556,7 @@ def _make_request(
             # HTTP version
             http_version,
             response.status,
-            response.length_remaining,  # type: ignore[attr-defined]
+            getattr(response, "length_remaining", None),
```

Comment:
Can we add support for `length_remaining` in this PR?

Reply:
Do you mean adding `self.length_remaining = 0` or actually handling it correctly?

Reply:
`self.length_remaining = 0` seems appropriate, the actual implementation can happen in the streaming issue.

Reply:
Done. By the way, If we keep `length_remaining` eventually, this means we should probably add it to `BaseHTTPResponse` and fix the mypy error in the process.

Reply:
Agreed should be a part of `BaseHTTPResponse`.

Reply:
Recorded here: https://github.com/urllib3/urllib3/issues/3315